### ISSUE TITLE
NoRisk Client Website https

### ISSUE THREAD

**Issue by OnlYKai** [HGLabor/client-issues#171]
Wenn der Browser das https:// am anfang nicht automatisch einfügt muss man des dazuschreiben wenn man auf die seite will ist aber glaube einstellungssache der Seite

**Comment by KeineLustAufName** [comment]
Und warum ist das jetzt ein Bug oder ein feature request oder so?

**Comment by WorldEnd0** [comment]
> Wenn der Browser das https:// am anfang nicht automatisch einfügt muss man des dazuschreiben wenn man auf die seite will ist aber glaube einstellungssache der Seite

Ich würde sagen, dass liegt an deinem Browser

**Comment by KeineLustAufName** [comment]
Dann schreib das https:// halt dazu HuhChamp

**Comment by KeineLustAufName** [comment]
https heißt halt das Website sicher ist ohne halt nicht (glaube ich) ohne https kannst du halt nicht auf die Seite. Schreib es einfach dazu

**Comment by WorldEnd0** [comment]
> https heißt halt das Website sicher ist ohne halt nicht (glaube ich) ohne https kannst du halt nicht auf die Seite. Schreib es einfach dazu

Right. http:// is unsicher und https:// meist sicher. Trotzdem blockieren manche Browser ansich .jar files und melden es als Unsicher
